components/MadeWithPrezly/MadeWithPrezly.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+
+import { LogoPrezly } from 'icons';
+
+import styles from './MadeWithPrezly.module.scss';
+
+const FOOTER_HEIGHT = 88;
+
+export function MadeWithPrezly() {
+    const [isMadeWithPrezlyVisible, setIsMadeWithPrezlyVisible] = useState(true);
+
+    useEffect(() => {
+        function madeWithPrezlyListener() {
+            setIsMadeWithPrezlyVisible(
+                window.innerHeight + Math.round(window.scrollY) <
+                    document.body.offsetHeight - FOOTER_HEIGHT,
+            );
+        }
+        if (typeof window !== 'undefined') {
+            window.onscroll = madeWithPrezlyListener;
+        }
+
+        return () => {
+            if (typeof window !== 'undefined') {
+                window.onscroll = null;
+            }
+        };
```

Comment:
This is not good, this way of attaching listeners can conflict with browser extensions or other code. You should use `target.addEventListener` to attach events.
Also, you don't need `typeof window` checks in `useEffect`, since it never runs on the server.
Also also, good idea to add `passive: true` to the listener options to indicate that this listener would not interfer with scrolling.

```suggestion
        window.addEventListener('scroll', madeWithPrezlyListener, { passive: true });

        return () => {
            window.removeEventListener('scroll', madeWithPrezlyListener, { passive: true });
        };
```